anafi:
  anafi:
    ros__parameters:
      autorecord: false  			# Enable autorecord at takeoff
      banked_turn: false  			# Enable banked turn
      camera_mode: 0  				# Camera mode: 0 = camera in recording mode, 1 = camera in photo mode
      camera_operated: false  			# Commands are relative to the camera pitch
      download_folder: ~/Pictures/Anafi 	# Absolute path to the download folder
      ev_compensation: 9  			# EV compensation: 0 = -3.00 EV; 3 = -2.00 EV; 6 = -1.00 EV; 9 = 0.00 EV; 12 = 1.00 EV; 15 = 2.00 EV; 18 = 3.00 EV
      hdr: true  				# Enable HDR
      home_type: 4  				# Home type for RTH: 1 = return to the last takeoff location; 3 = return to a user-set custom location; 4 = return to the pilot position
      hovering_altitude: 10.0  			# RTH ending hovering altitude (in m) [1.0, 10.0]
      image_style: 0  				# Images style: 0 = natural look style; 1 = produces flat and desaturated images, best for post-processing; 2 = intense style: bright colors, warm shade, high contrast; 3 = pastel style: soft colors, cold shade, low contrast
      max_altitude: 2.0  			# Max altitude (in m) [0.5, 4000.0]
      max_distance: 10.0  			# Max distance (in m) [10.0, 4000.0]
      max_gimbal_speed: 180.0  			# Max gimbal speed (in deg/s) [1.0, 180.0]
      max_horizontal_speed: 1.0  		# Max horizontal speed (in m/s) [0.1, 15.0]
      max_pitch_roll_rotation_speed: 200.0  	# Max pitch/roll rotation speed (in deg/s) [40.0, 300.0]
      max_tilt: 10.0  				# Max pitch/roll (in deg) [1.0, 40.0]
      max_vertical_speed: 1.0  			# Max vertical speed (in m/s) [0.1, 4.0]
      max_yaw_rotation_speed: 180.0  		# Max yaw rotation speed (in deg/s) [3.0, 200.0]
      max_zoom_speed: 10.0  			# Max zoom speed (in tan(deg)/sec) [0.1, 10.0]
      precise_home: true  			# Enable precise RTH
      rth_autotrigger: true  			# Enable auto trigger RTH
      rth_ending_behavior: 1  			# Ending behavior for RTH: 0 = land after RTH; 1 = hover after RTH
      rth_min_altitude: 20.0  			# RTH minimum altitude (in m) [20.0, 100.0]
      streaming_mode: 0  			# Streaming mode: 0 = minimize latency with average reliability (best for piloting); 1 = maximize the reliability with an average latency; 2 = maximize the reliability using a framerate decimation with an average latency
